src/components/Button/Button.tsx

```diff
@@ -5,15 +5,26 @@ type buttonProps = {
   buttonText: string;
   buttonClick?: () => void;
   icon?: React.ReactNode;
+  cartButton?: true;
   size: "small" | "medium" | "large";
 };
 
-function Button({ buttonText, buttonClick, icon, size }: buttonProps) {
+function Button({
+  buttonText,
+  buttonClick,
+  icon,
+  size,
+  cartButton,
+}: buttonProps) {
   const { cartQuantity } = useCart();
 
   return (
     <button onClick={buttonClick} className={size}>
-      <span>{cartQuantity === 0 ? "" : cartQuantity} </span>
+      {cartButton ? (
```

Comment:
if you wanted to shorten it, you could use the same syntax you used below for icon

`			{cartButton && <span>{cartQuantity === 0 ? '' : cartQuantity} </span>}`